postgresql/doc/Postgresql.md

```diff
@@ -0,0 +1,48 @@
+# Postgresql plugin documentation
```

Comment:
I would point somewhere here to the documentation of credentials and give instruction that credentials must be set before using any of these function/procedure.